.github/CODEOWNERS

```diff
@@ -729,14 +729,14 @@
 # ServiceOwners:                                     @QingChenmsft @vaishnavk @juhacket
 
 # PRLabel: %WebPubSub
-/sdk/webpubsub/azure-messaging-webpubsub-client/     @vicancy
-/sdk/webpubsub/azure-messaging-webpubsub/            @vicancy
+/sdk/webpubsub/azure-messaging-webpubsub-client/     @vicancy @chenkennt @zackliu
```

Comment:
@zackliu needs to be marked as a public member of the Azure organization

Reply:
Membership updated